alma/lib/Model/PaymentData.php

```diff
@@ -175,7 +176,26 @@ public static function dataFromCart($cart, $context, $feePlans, $forPayment = fa
             $customerData['business_name'] = $billingAddress->company;
         }
 
+        $carrierHelper = new CarrierHelper($context);
+        $cartHelper = new CartHelper($context);
+
+        $websiteCustomerDetails = [
+            'new_customer' => self::getNewCustomer($customer->id),
+            'is_guest' => (bool) $customer->is_guest,
+            'created' => strtotime($customer->date_add),
+            'current_order' => [
+                'purchase_amount' => almaPriceToCents($purchaseAmount),
+                'payment_method' => 'alma',
```

Comment:
is there a `constant` or `method` where we could retrieve the payment method name instead of writing `'alma'` here ?

Reply:
No I didn't find a constant or method to retrieve the payment method :/ 
But is illogical, the payment method is inevitably Alma because is the module alma.

Reply:
I would like to say that this string should be stored in a constant (called `PAYMENT_METHOD` for example) and reused everywhere with this constant... this could be done into this big refactorisation PR 😉 ... but we also could add a clickup `debt` ticket for this subject 😘

Reply:
You're right, I did that in this PR :)